statsmodels/survey/tests/test_summary_stats.py

```diff
@@ -0,0 +1,35 @@
+import summary_stats as ss
+import pandas as pd
+import numpy as np
+from numpy.testing import (assert_almost_equal, assert_equal, assert_array_less,
+                           assert_raises, assert_allclose)
+
+strata = np.r_[1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0]
+cluster = np.r_[0, 0, 2, 2, 3, 3, 4, 4, 4, 6, 6, 6]
+weights = np.r_[1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2].astype(np.float64)
+data = np.asarray([[1, 3, 2, 5, 4, 1, 2, 3, 4, 6, 9, 5],
+                   [5, 3, 2, 1, 4, 7, 8, 9, 5, 4, 3, 5]], dtype=np.float64).T
+design = ss.SurveyDesign(strata, cluster, weights)
+
+def test_design():
+    assert_equal(design.ncs, np.array([2, 3]))
+    assert_equal(design.sfclust, np.array([0, 0, 1, 1, 1]))
+    assert(len(np.unique(design.sclust)), len(np.unique(cluster)))
```

Comment:
Avoid Python assert, use Numpy's `assert_equal` here as well (works better in CI tools).